sysvabi64/sysvabi64.rst

```diff
@@ -1435,10 +1433,20 @@ HWCAP_ flags.
     unsigned long _size; /* Size of the struct, so it can grow.  */
     unsigned long _hwcap;
     unsigned long _hwcap2;
+    unsigned long _hwcap3;
+    unsigned long _hwcap4;
   };
 
   typedef struct __ifunc_arg_t __ifunc_arg_t;
 
+The ``_IFUNC_ARG_HWCAP`` bit will be set in the first parameter when the
+second parameter is available. When this is the case, the second parameter
+is a pointer to a ``__ifunc_arg_t`` object and its ``_size`` field is set
+to ``sizeof(__ifunc_arg_t)``. IFUNC resolver functions can use the value
+of the ``_size`` field to check if additional ``_hwcap`` fields are available.
+When the ``_IFUNC_ARG_HWCAP`` bit is not set, only one parameter is passed
```

Comment:
(this makes it sound like the first parameter is _only_ _IFUNC_ARG_HWCAP in the "modern" case, whereas in actual fact ifuncs that only need AT_HWCAP can always just check the bits in the first argument and ignore the second entirely.)